### DISCUSSION TITLE
Xbox 360 Controller for Windows does not show up as an input device

### DISCUSSION THREAD

**Discussion by Emreld3000** [KeyPuncher/WiinUPro#174] [Help]
My Wii U Pro Controller was registered as a Xbox 360 Controller. But, it doesn't not show up as an input device and instead shows up as an Other Device, so I can't use this. How can I get it registered as an input device?

**Comment by actopozipc** [comment]
Are you sure you cant use it anyway? It also shows me as Other Device, it still gets detected by steam. Only downside might be that I have to configure it everytime as default input in the game, but this might be a FIFA bug as well

**Reply by Emreld3000** [reply]
Yes, I am sure that I could not use it anyway.

Eventually, I did find a way to get it to show up as an input device. There was some order of connecting it and forgetting old devices that I was unclear on. However, steam still fails to recognize it, so it's useless. I don't really have any more information as to why steam doesn't recognize it as an xbox controller, but it just... doesn't.

**Reply by KeyPuncher** [reply]
I think Steam can use the controllers without needing conversion. At least I know it works on Steam Link, and I think Steam Deck as well, so it should be within Steam for Windows too.

**Reply by Emreld3000** [reply]
Would you mind if I gave you a more detailed summary of how exactly it reacts in the morning? I would love to tell you what exactly is happening right now, but it is currently too late for me to keep going.

**Reply by Emreld3000** [reply]
Hi Keypuncher,

Here's a summary of what I'm doing so we can figure out what is going wrong.

My steps for connecting the Wii U Pro Controller are as follows:
1. Make sure the Nintendo RVL-CNT-1-01-UC is forgotton, but not the "Xbox 360 Controller for Windows".
2. Activate Sync with WiinUSoft, and press the red button on the controller
3. Switch the device over to Xbox recognition.  I can see the device recognized in Device Manager as Xbox 360 Controller for Windows.

I can see in Config Window all my buttons are mapped.

When I got to Steam -> Steam Settings -> Controller, I can see Nintendo RVL-CNT-01-TR there. Not UC, TR.
However, if I attempt to begin setup, no buttons are recognized.

Steam Input for Xbox Controllers and Generic Controllers is on, but input for Switch Pro Controllers is off.

I also can't set it up if I keep the controller on the Detected Devices side of WiiUSoft.

**Reply by Emreld3000** [reply]
I should also note that, when recognized as an Xbox controller, the device DOES work on, say, Fightcade2. So the issue seems to be specific to Steam.

**Reply by PiplupPalace** [reply]
has there been any updates on this?

**Comment by KeyPuncher** [comment]
Windows 11?

**Reply by Emreld3000** [reply]
Yes, Windows 11.
